domain-cc/README.md

```diff
@@ -25,6 +28,22 @@ pip3 install -r requirements.txt
 uvicorn api:app --reload
 ```
 
+## testing it all together
+Get the java project up and running
+See [Local Setup](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Local-Setup) on the wiki.
```

Comment:
You probably don't need to run all the default containers. You can set `COMPOSE_PROFILES=""` -- [related Slack](https://dsva.slack.com/archives/C04QLHM9LR0/p1681149965333259?thread_ts=1681144545.160509&cid=C04QLHM9LR0).
Later, you can set `COMPOSE_PROFILES="cc"` after adding a container to `app/src/docker/docker-compose.yml`.

Reply:
I will submit follow-up PR to dockerize the code